features/ftue/impl/src/main/kotlin/io/element/android/features/ftue/impl/FtueFlowNode.kt

```diff
@@ -135,6 +147,9 @@ class FtueFlowNode @AssistedInject constructor(
             FtueStep.MigrationScreen -> {
                 backstack.newRoot(NavTarget.MigrationScreen)
             }
+            FtueStep.NotificationsOptIn -> {
+                backstack.newRoot(NavTarget.NotificationsOptIn)
+            }
```

Comment:
Since the new screen is displayed after `WelcomeScreen`, maybe move this case below the `WelcomeScreen` case, just for code clarity. Definitely not a blocker!